packages/cli/src/lib/cmds/init.ts

```diff
@@ -77,6 +79,15 @@ async function confirmTwaConfig(twaManifest: TwaManifest, prompt: Prompt): Promi
       validateDisplayMode,
   );
 
+  // TODO(andreban): Remove the alpha check when androidx.browser becomes stable.
+  if (twaManifest.alphaDependencies) {
```

Comment:
So where does this get set? What does a user have to do to get this prompt to appear?

Reply:
Adding `--alphaDependencies` when running `init` does it. Will add to the docs.

Reply:
done.